PLAYER/GetPlayersLastVehicle.md

```diff
@@ -8,8 +8,11 @@ ns: PLAYER
 Vehicle GET_PLAYERS_LAST_VEHICLE();
 ```
 
-```
-Alternative: GET_VEHICLE_PED_IS_IN(PLAYER_PED_ID(), 1);  
-```
+### Warning
```

Comment:
Are you able to add additional sections like this? I thought it would break native gen

Reply:
Yes 😀
https://github.com/citizenfx/natives/pull/801#discussion_r955428610